notebooks/1.2-pyc-eda-mix.py

```diff
@@ -0,0 +1,500 @@
+#        Oth	Other
+
+multiple_ecdf('SaleType')
+
+
+# **Insights**
+# 
+# - have a lot of categories, but appears that just the category New increase in the sale
+# - Maybe we should try an unsupervised model to join these categories
+
+# ## Sale date
+# 
+# Month and Year of the sale
+
+sns.pairplot(train[['SalePrice','MoSold','YrSold']])
+
+
+train[train['YrSold']==2010]['MoSold'].value_counts()
+
+
+fig = make_subplots(rows=1, cols=2,
+                   subplot_titles=("Mean", "Std"))
+
+fig.add_trace(go.Scatter(x=np.sort(train['YrSold'].unique()), 
+                         y=train.groupby(['YrSold'])['SalePrice'].mean()),
+            row=1,col=1)
+
+fig.add_trace(go.Scatter(x=np.sort(train['YrSold'].unique()), 
+                         y=train.groupby(['YrSold'])['SalePrice'].std()),
+            row=1,col=2)
+
+fig.update_layout(showlegend=False, title_text="Mean and Std of Sales Price per Year")
+fig.show()
+
+
+train['day'] = 1
+train['dateSold'] = pd.to_datetime(train.rename(columns={"YrSold": "year",
+                                                         "MoSold": "month"})[['year','month','day']],
+                                   unit='D')
+train.drop(columns=['day'], inplace=True)
+fig = px.line(train.groupby(['dateSold'])[['SalePrice', 'dateSold']].mean(), 
+              x=train.groupby(['dateSold'])[['SalePrice', 'dateSold']].mean().index,
+              y='SalePrice')
+fig.show()
+
+
+# **Insights**
+# - The last month we have just 6 data, so, probably, we don't have data of the full month
+# - Maybe it's better to join the 2 columns to understand the timeline
+# - The timeline appears to make more sense that the Month or Year column alone
+
+# ## Neighborhood Variables
+# 
+# - MSZoning
+# - Street
+# - Alley
+# - Neighborhood
+# - Condition 1
+# - Condition 2
+
+# ## MSZoning
+# 
+# Identifies the general zoning classification of the sale.
+# 		
+#        A	Agriculture
+#        C	Commercial
+#        FV	Floating Village Residential
+#        I	Industrial
+#        RH	Residential High Density
+#        RL	Residential Low Density
+#        RP	Residential Low Density Park 
+#        RM	Residential Medium Density
+
+multiple_ecdf('MSZoning')
+
+
+# **Insights**
+# 
+# - The RH category can be joined with the RM one
+# - C category have few data, but this graphic doesn't give us an option to join with another column
+
+# ## Street
+# 
+# 
+# Type of road access to property
+# 
+#        Grvl	Gravel	
+#        Pave	Paved
+
+train['Street'].value_counts()
+
+
+# **Insights**
+# 
+# - Few values aren't equal to Pave, so maybe we should just drop this column
+
+# ## Alley
+# 
+# Type of alley access to property
+# 
+#        Grvl	Gravel
+#        Pave	Paved
+#        NA 	No alley access
+
+train['Alley'].value_counts()
+
+
+multiple_ecdf('Alley')
+
+
+# **Insights**
+# 
+# - Almost all values are Nan
+# - The not nan variables appears to be very important
+# - I think that we should just fill the nan values in this column
```

Comment:
Wich is the kind of this data ? How should we fill the NAN values ?

Reply:
I was suggesting to join all values not nan in this column